CHANGELOG.md

```diff
@@ -10,26 +10,61 @@ CHANGELOG
 - Add support for Centos 7 ARM.
 - Add support for FSx Lustre DataCompressionType feature.
 - Add validation to prevent using a `cluster_resource_bucket` that is in a different region than the cluster.
+- Make sure slurmd service is not enabled before finalize stage, which will prevent user from unintentionally making compute node available in post-install process.
+- Change to ssh_target_checker.sh syntax that makes the script compatible with pdsh.
+- Install SSM agent on CentOS 7 and 8.
+- Add support for `security_group_id` in packer custom builders.
+- SGE: make `qstat` command in nodewatcher more robust in case a custom DHCP option set is configured.
 - Transition from IMDSv1 to IMDSv2.
 
 **CHANGES**
 
 - Ubuntu 16.04 is no longer supported.
 - Amazon Linux is no longer supported.
-- Upgrade Slurm to version 20.11.7.
-  - Add new SlurmctldParameters, power_save_min_interval=30, so power actions will be processed every 30 seconds
-  - Specify instance GPU model as GRES GPU Type in gres.conf, instead of previous hardcoded value for all GPU, Type=tesla
-- Upgrade Arm Performance Libraries (APL) to version 21.0.0
 - Make `key_name` parameter optional to support cluster configurations without a key pair.
 - Remove support for Python versions < 3.6.
-- Remove dependency on `future` package and `__future__` module.
+  - Remove dependency on `future` package and `__future__` module.
 - Root volume size increased from 25GB to 35GB on all AMIs. Minimum root volume size is now 35GB.
-- Add sanity check to prevent cluster creation in non officially supported AWS regions
+- Add sanity check to prevent cluster creation in non officially supported AWS regions.
```

Comment:
Prevent cluster creation in an AWS region not officially supported by ParallelCluster

Reply:
Done